src/Sql/NotificationCenter/dbo/Stored Procedures/Notification_DeleteById.sql

```diff
@@ -0,0 +1,10 @@
+CREATE PROCEDURE [dbo].[Notification_DeleteById]
```

Comment:
⚠️ I don't see this in C# anywhere and don't know why we'd need it.

Reply:
Fixed in 006f1b4